src/Compilers/CSharp/Portable/CSharpParseOptions.cs

```diff
@@ -21,10 +21,15 @@ public sealed class CSharpParseOptions : ParseOptions, IEquatable<CSharpParseOpt
         private ImmutableDictionary<string, string> _features;
 
         /// <summary>
-        /// Gets the language version.
+        /// Gets the effective language version.
         /// </summary>
         public LanguageVersion LanguageVersion { get; private set; }
 
+        /// <summary>
+        /// Gets the specified language version.
+        /// </summary>
+        public LanguageVersion SpecifiedLanguageVersion { get; private set; }
```

Comment:
Thanks. Consider adding this information into the doc comment. Also, consider if name "RequestedLanguageVersion" would be better for this property. 

---
In reply to: [74466561](https://github.com/dotnet/roslyn/pull/12969#discussion_r74466561) [](ancestors = 74466561)